tsuru/admin/node_test.go

```diff
@@ -705,3 +707,103 @@ func (s *S) TestRebalanceNodeCmdRunGivingUp(c *check.C) {
 	c.Assert(err, check.IsNil)
 	c.Assert(stdout.String(), check.Equals, "Are you sure you want to rebalance containers? (y/n) Abort.\n")
 }
+
+func (s *S) TestInfoNodeCmdRun(c *check.C) {
+	var buf bytes.Buffer
+	context := cmd.Context{Args: []string{"host1:2375"}, Stdout: &buf}
+	trans := &cmdtest.ConditionalTransport{
+		Transport: cmdtest.Transport{Message: `{
+	"node":{"Address":"host1.com:2375","IaaSID":"test123","Metadata":{"foo": "bar"},"Status":"enabled","Pool":"pool1","Provisioner":"fake"},
+	"status":{
+		"Address":"host1.com:2375","LastSuccess":"2017-12-06T16:55:26.178-02:00","LastUpdate":"2017-12-06T16:55:26.178-02:00",
+		"Checks":[{"Time":"2017-12-06T16:55:26.178-02:00","Checks":[{"Name":"ok1","Err":"","Successful":true},{"Name":"ok2","Err":"","Successful":true}]}]
+	},
+	"units":[{"ID":"a834h983j498j","Name":"","AppName":"fake","ProcessName":"","Type":"","Status":"","HostAddr":"host1.com","HostPort":"2375","IP":"",
+			"Address": {"Scheme":"","Opaque":"","User":null,"Host":"host1.com:2375","Path":"","RawPath":"","ForceQuery":false,"RawQuery":"","Fragment":""}}
+	]
+}`, Status: http.StatusOK},
+		CondFunc: func(req *http.Request) bool {
+			return req.URL.Path == "/1.6/node/host1:2375"
+		},
+	}
+	statusT, _ := time.Parse(time.RFC3339, "2017-12-06T16:55:26.178-02:00")
+	statusTStr := statusT.Local().Format(time.Stamp)
+	manager := cmd.Manager{}
+	client := cmd.NewClient(&http.Client{Transport: trans}, nil, &manager)
+	err := (&InfoNodeCmd{}).Run(&context, client)
+	c.Assert(err, check.IsNil)
+	expected := `+----------------+---------+-------+-------------+------------------+
```

Comment:
Does this need to be a table? We'll always have a single entry, so maybe we should simply list the values like we do with `app-info` and `event-info` and have a table only for the metadata, units and node statuses. Something like:
```
Address: host1.com:2375
Status: enabled
IaaS ID: test123
Pool: pool1
Provisioner: fake
Metadata:
+-----+--------+
| key | value  |
+-----+--------+
| foo | bar    |
+-----+--------+
...
```